ssv/spectest/tests/runner/postconsensus/invalid_then_quorum.go

```diff
@@ -0,0 +1,183 @@
+package postconsensus
+
+import (
+	"fmt"
+
+	"github.com/attestantio/go-eth2-client/spec"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// InvalidThenQuorum  tests a runner receiving an invalid message then a valid quorum, terminating successfully
+func InvalidThenQuorum() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+	expectedError := "failed processing post consensus message: invalid post-consensus message: could not verify Beacon partial Signature: wrong signature"
+	multiSpecTest := &tests.MultiMsgProcessingSpecTest{
+		Name: "post consensus invalid then quorum",
+		Tests: []*tests.MsgProcessingSpecTest{
+			{
+				Name: "sync committee contribution",
+				Runner: decideRunner(
+					testingutils.SyncCommitteeContributionRunner(ks),
+					&testingutils.TestingSyncCommitteeContributionDuty,
+					testingutils.TestSyncCommitteeContributionConsensusData,
+				),
+				Duty: &testingutils.TestingSyncCommitteeContributionDuty,
+				Messages: []*types.SSVMessage{
+					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PostConsensusWrongSigSyncCommitteeContributionMsg(ks.Shares[1], 1, ks)),
+
+					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[1], 1, ks)),
+					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[2], 2, ks)),
+					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[3], 3, ks)),
+				},
+				PostDutyRunnerStateRoot: invalidThenQuorumSyncCommitteeContributionSC().Root(),
+				PostDutyRunnerState:     invalidThenQuorumSyncCommitteeContributionSC().ExpectedState,
+				OutputMessages:          []*types.SignedPartialSignatureMessage{},
+				BeaconBroadcastedRoots: []string{
+					testingutils.GetSSZRootNoError(testingutils.TestingSignedSyncCommitteeContributions(testingutils.TestingSyncCommitteeContributions[0], testingutils.TestingContributionProofsSigned[0], ks)),
+					testingutils.GetSSZRootNoError(testingutils.TestingSignedSyncCommitteeContributions(testingutils.TestingSyncCommitteeContributions[1], testingutils.TestingContributionProofsSigned[1], ks)),
+					testingutils.GetSSZRootNoError(testingutils.TestingSignedSyncCommitteeContributions(testingutils.TestingSyncCommitteeContributions[2], testingutils.TestingContributionProofsSigned[2], ks)),
+				},
+				DontStartDuty: true,
```

Comment:
why not start duty?

Reply:
The decided runner created has the state necessary to process post-consensus msgs (e.g. an instance with decide = True and a decided value). Starting the duty would overwrite this state. I've used this approach since it's the one used in other tests in this folder.